private_gpt/settings/settings.py

```diff
@@ -82,6 +82,7 @@ class DataSettings(BaseModel):
 
 class LLMSettings(BaseModel):
     mode: Literal["local", "openai", "sagemaker", "mock"]
+    max_new_tokens: int
```

Comment:
I believe the default should be defined in this file, instead of `settings.yaml`.
Could you also add the description of this parameter next to the default? C.f. below

```suggestion
    max_new_tokens: int = Field(256, description="The maximum number of token that the LLM is authorized to generate in one completion.")
```